app/perf_overlay.conf

```diff
@@ -1,7 +1,10 @@
 CONFIG_PERFORMANCE_COUNTERS=y
 CONFIG_PERFORMANCE_COUNTERS_COMPONENT=y
-CONFIG_PERFORMANCE_COUNTERS_LL_TASKS=y
+# disable ll task level statistics to reduce logging overhead
+#CONFIG_PERFORMANCE_COUNTERS_LL_TASKS=y
```

Comment:
BTW, perf_overlay.conf is for performance monitoring, not for performance enhancement (why would you not have enhanced performance in your default configuration), right? The name sounds potentially a bit confusing, can we rename it? Or at least add a comment at the top

Reply:
I'll let @singalsu  comment, I think he's the only known user at the moment for this. Not sure if we have some Ci jobs somewhere that have the name hardcoded -- not sure worth the hassle TBH.

Reply:
No problem to rename it. I don't think we have CI builds and tests with it. It was planned but our plans got changed.

Reply:
This is fine as it is, this feature is for performance monitoring but it can have impact on peak when logging.